analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -390,11 +391,19 @@ class Pub(
         if (!definitionFile.isFile) return emptyList()
 
         return analyzerResultCacheAndroid.getOrPut(packageName) {
+            val packageManagers = analyzerConfig.packageManagers?.toMutableMap() ?: mutableMapOf()
+            val gradleConfiguration = packageManagers["Gradle"] ?: PackageManagerConfiguration()
+            val gradleOptions = gradleConfiguration.options?.toMutableMap() ?: mutableMapOf()
+            val gradleVersion = gradleOptions.getOrPut("gradleVersion") { GRADLE_VERSION }
```

Comment:
`getOrPut` is not really necessary. Consider using `gradleOptions["gradleVersion"]`.

Reply:
Why is it not necessary? If no `"gradleVersion"` is present yet (that should take precedence over `GRADLE_VERSION`), we need to somehow put `GRADLE_VERSION` into the map.